modules/oc-mirror-about-cache-and-workspace-dirs.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oc-mirror-about-cache-and-workspace-dirs_{context}"]
+= About the --cache-dir and --workspace flags
+
+You can use the `--cache-dir` flag to specify a directory for storing a persistent cache of image metadata, manifests, and the history of previous mirroring operations. You can use this cache directory to perform incremental mirroring and avoid remirroring unchanged images. 
```

Comment:
@sr1kar99 - the section "_and the history of previous mirroring operations_" , could you maybe just remove this altogether , its not really  history we store. We store all image blobs, and manifests